main.go

```diff
@@ -352,7 +352,8 @@ func favorites(cCtx *cli.Context) error {
 			return helper.CreateSubShell(favorite.Account, favorite.Name, favorite.CAR, stak)
 		}
 	} else {
-		car, err := kion.GetCARByName(cCtx.String("endpoint"), cCtx.String("token"), favorite.CAR)
+		// If access type is specifically set to web
+		car, err := kion.GetCARByNameAndAccount(cCtx.String("endpoint"), cCtx.String("token"), favorite.CAR, favorite.Account)
 		if err != nil {
 			return err
 		}
```

Comment:
Try reverting to the old func call `kion.GetCARByName` then below the error checking add:

```
		car.AccountNumber = favorite.AccountNumber
```

Reply:
Confirmed this works.

Reply:
This may work, but this seems like a poor workaround against a malfunctioning API.  I saw you had a workaround that used a private API.  Do you think we should use the private API until the public one is fixed?

Reply:
I'd rather not since this workaround is very minimal and we're dealing with known information stored in the favorite. It will at least be more stable than the private api.